### ISSUE TITLE
[FR] Map feature

### ISSUE THREAD

**Issue by collateral87** [sander1988/Indego#249]
Thank you for developing this awsome intergration, it is a blessing for all Indego owners.

I followed the development and saw that there were already several approaches to providing a map. Since then, I hope that this feature will come. Unfortunately, all solutions were discarded again or no further development took place. I'm not a developer, unfortunately, so there's little I can do. But I wanted to ask if this feature would be technically feasible at all or not?

**Comment by Andi1969** [comment]
Würde mich auch über eine Karte freuen

**Comment by kimzeuner** [comment]
You are right, having the map within HA would be great but as mentioned in the past we have to be careful not to stress the Bosch API too much. I'm testing a map feature for some time now and maybe i will create a new PR for that in the next weeks. 

@Luiun16 Am i right that you mean the location that can be configured in the Bosch App (adress where the Indego is located) ? I think just reading the configured location should be easy and i will try that in the next days. I think configuring the location in HA will not be possible but i will also check that.

**Comment by Gerrit1309** [comment]
hey there, 
Check out this old video, perhaps it can help you to code it for everyone! 
https://www.youtube.com/watch?v=-sOSOXz-Vv0
https://github.com/SmarthomeAddicted/Indego-Integration

I would like to have a card with the Position of the mower, too! 

Thanks a lot!

**Comment by Luiun16** [comment]
@kimzeuner Exactly, I mean the position of the Indego. In general, the app displays the approximate outline of my garden. When the Indego is moving, I can see this on the map by means of a yellow dot that is updated every X seconds. 

Both points (outline of the garden or map and the position of the Indego) would be an exciting added value.

**Comment by kimzeuner** [comment]
Ok, understood. I thought you meant the location of your home but you mean the position of the mower in your garden (as displayed in the bosch app). I will do my best

**Comment by Luiun16** [comment]
Hello @kimzeuner ,

it's been a while since we last discussed the development of the "map feature". I just wanted to ask if the development is still your plan or if there is anything new in general.

Thank you very much!

Best regards 
Dennis

**Comment by kimzeuner** [comment]
Hello,
Yes, i`m still working on it but due to early winter sleep of my mower and less time i was not able to finalize it. Will give it a new try within the next weeks. 

Regards!

**Comment by WhyLev** [comment]
🤖 **AI Summary of the solution:**

The user inquired about the feasibility of developing a map feature for the Indego integration, which has seen multiple discarded attempts, and received confirmation from a developer that work is still ongoing despite delays.
